modules/test/playwright/pages/dynamic-data-mapping-form-web/FormBuilderSidePanelPage.ts

```diff
@@ -36,6 +37,7 @@ export class FormBuilderSidePanelPage {
 		});
 		this.allowGuestUsers = page.getByLabel('Allow Guest Users to Send');
 		this.backButton = page.getByRole('button', {name: 'Back'});
+		this.helpTextField = page.getByLabel('Help Text');
```

Comment:
Hi @larissacribeiro, as you told me, I agree that the help text is not related to the test case. So we can stop using it from now on.

Reply:
done!